debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/J9BCUtil.java

```diff
@@ -708,6 +708,8 @@ private static void dumpClassJ9ExtraModifiers(PrintStream out, long accessFlags)
 			out.append("(preverified) ");
 		if ((accessFlags & J9JavaAccessFlags.J9AccClassAnonClass) != 0)
 			out.append("(anonClass) ");
+		if ((accessFlags & J9JavaAccessFlags.J9AccClassIsUnmodifiable) != 0)
```

Comment:
This will fail with `NoSuchFieldError` when accessing a core file generated prior to the addition of `J9AccClassIsUnmodifiable`.

This should use a static field initialized with
> J9ConstantHelper.getLong(J9JavaAccessFlags.class, "J9AccClassIsUnmodifiable", 0)

Reply:
The bit won't actually be set in older cores. Should all of the flag accesses not be using the new mechanism?

Reply:
It's fine to use fields of the generated structure types when the constant has always been around; it's only when new ones are added that we need to be careful.

This got me thinking that perhaps there's a more general way to handle this, say, by introducing a way to cause `J9DDRClassLoader` to include new fields even for old core files with a suitable default value.

Reply:
Constant fix is in https://github.com/eclipse/openj9/pull/5118

Further discussions can take place elsewhere.